.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
Hmm...

I think that we should support (and therefore test) any active, LTS, or maintenance version of node (less `v0.10`). That would mean `v0.11`, `v0.12`, all releases of iojs, and node 4/5.

I doubt it would slow down travis that much, but if we really wanted to make it run fast, we could enable travis [fast finishing](https://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing).